packages/scandipwa/src/component/Field/Field.component.js

```diff
@@ -203,16 +203,18 @@ export class Field extends PureComponent {
             onChange: onChange || noopFn
         };
 
+        const disableCheckbox = isDisabled ? !isDisabled : null;
```

Comment:
Won't just putting isDisabled work for checked attribute ?

Reply:
If I leave isDisabled, default checked option wouldn't be checked and stay unchecked even when trying to select them.
Like in this video https://vimeo.com/674792314/a12b76c701.

If left as is: https://vimeo.com/674792819/f61fc2b5db